src/components/UriViewer.js

```diff
@@ -99,17 +103,33 @@ const PreviewContent = ({ uri, metadata, metadata: { bucket, name }, googleProje
   ])
 }
 
-const DownloadButton = ({ uri, metadata: { bucket, name, fileName, size } }) => {
+const DownloadButton = ({ uri, metadata: { bucket, name, fileName, size }, accessUrl }) => {
   const signal = Utils.useCancellation()
   const [url, setUrl] = useState()
   const getUrl = async () => {
-    try {
-      const { url } = await Ajax(signal).Martha.getSignedUrl({ bucket, object: name, dataObjectUri: isGs(uri) ? undefined : uri })
-      const workspace = workspaceStore.get()
-      const userProject = await getUserProjectForWorkspace(workspace)
-      setUrl(knownBucketRequesterPaysStatuses.get()[bucket] ? Utils.mergeQueryParams({ userProject }, url) : url)
-    } catch (error) {
-      setUrl(null)
+    if (accessUrl && accessUrl.url) {
```

Comment:
Can you use `accessUrl?.url`?

Reply:
Meybe. What version of ES does terra-ui use? And how can one tell?

Reply:
https://create-react-app.dev/docs/supported-browsers-features
This was added in https://github.com/facebook/create-react-app/releases/tag/v3.3.0
Note, though, that it [should work in most browsers](https://caniuse.com/mdn-javascript_operators_optional_chaining), which is really what matters.